src/stories/AppContentSwitcher.stories.jsx

```diff
@@ -0,0 +1,74 @@
+import AppContentSwitcher from '~/components/app-content-switcher/AppContentSwitcher'
+import { useState } from 'react'
+import { action } from '@storybook/addon-actions'
```

Comment:
Set the correct order of imports

Reply:
+

Reply:
And set it like this: 
import { useState } from 'react'
import { action } from '@storybook/addon-actions'

import AppContentSwitcher from '~/components/app-content-switcher/AppContentSwitcher'